src/rgw/rgw_crypt.cc

```diff
@@ -975,7 +975,7 @@ static int get_sse_s3_bucket_key(req_state *s,
       if (res != -ECANCELED) {
         break;
       }
-      res = s->bucket->try_refresh_info(s, nullptr);
+      res = s->bucket->try_refresh_info(s, nullptr, null_yield);
```

Comment:
you can use `s->yield` here and anywhere else there is a `req_state` which is commonly called `s`.